app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from .cafe import Cafe
+from .errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: "Cafe") -> None:
```

Comment:
The function should return a string as specified in the task description. Currently, it only returns a string when exceptions are raised or masks need to be bought. Consider ensuring that a string is returned in all cases.